runhouse/resources/hardware/cluster_factory.py

```diff
@@ -338,7 +338,8 @@ def ondemand_cluster(
         autostop_mins (int, optional): Number of minutes to keep the cluster up after inactivity,
             or ``-1`` to keep cluster up indefinitely.
         use_spot (bool, optional): Whether or not to use spot instance.
-        image_id (str, optional): Custom image ID for the cluster.
+        image_id (str, optional): Custom image ID for the cluster. If using a docker image, please use the following
+            string format: "docker:<registry>/<image>:<tag>".
```

Comment:
Maybe we should link your new docs here?

Reply:
(will link in the guide pr)